Tasks/JenkinsDownloadArtifactsV1/package.json

```diff
@@ -11,8 +11,8 @@
     "artifact-engine": "0.1.26",
     "azure-pipelines-task-lib": "2.8.0",
     "azure-pipelines-tasks-azure-arm-rest": "1.0.0",
-    "azure-storage": "2.2.1",
-    "decompress-zip": "0.3.0",
+    "azure-storage": "^2.10.4",
```

Comment:
Can we remove the caret (^) symbol? It will automatically use any new minor or patch-level version, which might cause issues.

Reply:
@chandrasekharmudili, good point, removed `^` symbol for `azure-storage` and `decompress-zip` dependencies.